LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -305,3 +305,17 @@ void Utility::SanitizeJson(_Inout_ std::wstring& str)
         i++;
     }
 }
+
+/// <summary>
+/// Comparing wstrings with ignoring the case
+/// </summary>
+/// <param name="stringA"></param>
+/// <param name="stringB"></param>
+/// <returns></returns>
+bool Utility::CompareWStrings(wstring stringA, wstring stringB)

```

Comment:
Also got this neat way online 

    // Convert the strings to lowercase
    std::transform(str1.begin(), str1.end(), str1.begin(), ::towlower);
    std::transform(str2.begin(), str2.end(), str2.begin(), ::towlower);
    
    // Compare the lowercase strings
    int result = str1.compare(str2);

 std::transform() and ::towlower may be slightly faster because it does not involve additional conversions or memory allocations. However, this method only works with the current locale and may not produce the correct results for some Unicode characters that do not have a lowercase equivalent in the current locale.

No need to change your implementation. I was just intrigued by comparing wstrings in C++